stdlib/3/email/mime/text.pyi

```diff
@@ -1,8 +1,7 @@
 # Stubs for email.mime.text (Python 3.4)
 
-from typing import Optional
 from email.mime.nonmultipart import MIMENonMultipart
+from typing import Optional
 
 class MIMEText(MIMENonMultipart):
-    def __init__(self, _text: str, _subtype: str = ...,
-                 _charset: Optional[str] = ...) -> None: ...
+    def __init__(self, _text: str, _subtype: str = ..., _charset: Optional[str] = ...) -> None: ...
```

Comment:
Did you miss it here?

Reply:
Thanks, good catch. Missed it in another place as well :-)

Reply:
Looks like these are issues which stubtest already caught before #3835. I was fixing the delta, which is why I missed them.